source/worksheet/worksheet.cpp

```diff
@@ -361,11 +354,12 @@ void worksheet::active_cell(const cell_reference &ref)
 
     if (!primary_view.has_selections())
     {
-        primary_view.add_selection(selection());
+        primary_view.add_selection(selection(pane_corner::bottom_right, ref));
+    }
+    else
+    {
+        primary_view.selection(0).active_cell(ref);
```

Comment:
this doesn't seem right. If it has no selections, why does it appear to be accessing the 0 index?

Reply:
misread, this is the "already have selections" condition